README.md

```diff
@@ -10,57 +10,86 @@ Status](https://badge.buildkite.com/26d895f5525652e57915a607d0ecd3fc945c8280a0bd
+    - [Fetch source JARs](#fetch-source-jars)
+    - [Checksum verification](#checksum-verification)
+    - [Using a custom Coursier download url](#using-a-custom-coursier-download-url)
+    - [`artifact` helper macro](#artifact-helper-macro)
+    - [`java_plugin_artifact` helper macro](#java_plugin_artifact-helper-macro)
+    - [Multiple `maven_install` declarations for isolated artifact version trees](#multiple-maven_install-declarations-for-isolated-artifact-version-trees)
+    - [Detailed dependency information specifications](#detailed-dependency-information-specifications)
+    - [Artifact exclusion](#artifact-exclusion)
+    - [Compile-only dependencies](#compile-only-dependencies)
+    - [Test-only dependencies](#test-only-dependencies)
+    - [Resolving user-specified and transitive dependency version conflicts](#resolving-user-specified-and-transitive-dependency-version-conflicts)
+    - [Overriding generated targets](#overriding-generated-targets)
+    - [Proxies](#proxies)
+    - [Repository aliases](#repository-aliases)
+    - [Repository remapping](#repository-remapping)
+    - [Hiding transitive dependencies](#hiding-transitive-dependencies)
+    - [Accessing transitive dependencies list](#accessing-transitive-dependencies-list)
+    - [Fetch and resolve timeout](#fetch-and-resolve-timeout)
+    - [Ignoring empty jars](#ignoring-empty-jars)
+    - [Duplicate artifact warning](#duplicate-artifact-warning)
+    - [Provide JVM options for Coursier with `COURSIER_OPTS`](#provide-jvm-options-for-coursier-with-coursier_opts)
+    - [Resolving issues with nonstandard system default JDKs](#resolving-issues-with-nonstandard-system-default-jdks)
+    - [Exporting and consuming artifacts from external repositories](#exporting-and-consuming-artifacts-from-external-repositories)
+    - [Publishing to External Repositories](#publishing-to-external-repositories)
+- [Configuring the dependency resolver](#configuring-the-dependency-resolver)
+    - [Common options](#common-options)
+    - [Configuring Coursier](#configuring-coursier)
+    - [Configuring Maven](#configuring-maven)
+- [IPv6 support](#ipv6-support)
+- [Developing this project](#developing-this-project)
+    - [Verbose / debug mode](#verbose--debug-mode)
+    - [Tests](#tests)
+    - [Installing the Android SDK on macOS](#installing-the-android-sdk-on-macos)
+    - [Generating documentation](#generating-documentation)
 
 ## Features
 
+* MODULE.bazel bzlmod configuration (Bazel 6 and above only) 
 * WORKSPACE configuration
-* JAR, AAR, source JARs
+* Artifact version resolution with Coursier or Maven
+* Import downloaded JAR, AAR, source JARs
+* Export built JARs to Maven repositories
+* Pin resolved artifacts with their SHA-256 checksums into a version-controllable JSON file
 * Custom Maven repositories
-* Private Maven repositories with HTTP Basic Authentication
-* Artifact version resolution with Coursier
+* Private Maven repositories using HTTP Basic Authentication or `netrc`
```

Comment:
Do we want to encourage people to use basic authentication when Bazel itself won't? A simpler mental model of "if it works with Bazel, it'll work when resolving" might be better to lean into, and that suggests not mentioning basic auth....

Reply:
Fair point. Removed the basic auth mention.